src/status_im/ui2/screens/chat/components/reply/view.cljs

```diff
@@ -113,7 +114,9 @@
                                            (= constants/content-type-audio content-type))
                                    {:color (colors/theme-colors colors/neutral-50 colors/neutral-40)}))}
           (case (or content-type contentType)
-            constants/content-type-image   "Image"
+            constants/content-type-image   (if album-count
+                                             (str album-count \space "Images")
```

Comment:
Missing translation ;) Please, translate the words `Sticker` and `Audio` too if you can.

Also, consider pluralizing correctly, since `1 Images` is incorrect.

Reply:
I'll add the translations but the case that you mentioned ahould never happen because if the message has more than one photo it'll have the `album-count` property

Reply:
I don't thing a message can have more than one audio message or more than one sticker, but using i18n for them is essential